src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -250,6 +250,9 @@ void DefaultDACVerifier::VerifyAttestationInformation(const DeviceAttestationVer
         DeviceAttestationVendorReservedDeconstructor vendorReserved;
         ByteSpan certificationDeclarationPayload;
 
+        ChipLogProgress(Support, " ----- DefaultDACVerifier::VerifyAttestationInformation vendorId=0x%04X productId=0x%04X",
```

Comment:
Is this needed?

Reply:
doh, that was accidental, removed from this PR